dists/go-ipfs/go-ipfs-dist

```diff
@@ -79,8 +79,8 @@ dist_version() {
   cd "$cwd"
 
   # generate dist.json file
-  echo "---> generating dist.json"
-  ./mkdist.js "$version"
+  echo "---> generating config for $version"
+  ../../node_modules/.bin/gulp dist "--path=$(pwd)" "--version=$version"
```

Comment:
Using node means running `npm install` that's all users have to do. And when they have done that gulp is available locally and this works fine without anybody having to wonder about things. 

As for why using gulp, it makes implementing these things easier and I don't have to write the 100th argument parser for a node script